src/sandstorm/run-bundle.c++

```diff
@@ -1121,10 +1121,16 @@ private:
     KJ_SYSCALL(mount("/dev/urandom", "dev/urandom", nullptr, MS_BIND, nullptr));
     KJ_SYSCALL(mount("/dev/fuse", "dev/fuse", nullptr, MS_BIND, nullptr));
 
-    // Mount a tmpfs at /etc and copy over necessary config files from the host.
+    // Bind in the host's /etc as /etc.host.
+    // As noted in backup.c++, MS_BIND does not respect mount flags on the initial bind, and
+    // we have to issue a remount to set them.
+    KJ_SYSCALL(mount("/etc", "etc.host", nullptr, MS_BIND, nullptr));
+    KJ_SYSCALL(mount("/etc", "etc.host", nullptr, MS_BIND | MS_REMOUNT | MS_RDONLY, nullptr));
```

Comment:
Oh,  you'll want to throw `| MS_NOSUID | MS_NODEV | MS_NOEXEC` on here, otherwise this line will fail if these flags happen to be set on the source, because mount() thinks you're trying to remove the flags.